Lib/test/test_urlparse.py

```diff
@@ -644,8 +644,12 @@ def test_urlsplit_normalization(self):
         # bpo-36742: Verify port separators are ignored when they
         # existed prior to decomposition
         urlparse.urlsplit(u'http://\u30d5\u309a:80')
-        with self.assertRaises(ValueError):
+        with self.assertRaises(ValueError) as cm:
             urlparse.urlsplit(u'http://\u30d5\u309a\ufe1380')
+        self.assertEqual(str(cm.exception),
+                         "netloc u'\u30d7:80' contains invalid characters "
+                         "under NFKC normalization")
+        self.assertIsInstance(cm.exception.message, str)
```

Comment:
There is a `DeprecationWarning` in this line in CI https://travis-ci.org/python/cpython/jobs/543658580#L1921

```
/home/travis/build/python/cpython/Lib/test/test_urlparse.py:652: DeprecationWarning: BaseException.message has been deprecated as of Python 2.6
  self.assertIsInstance(cm.exception.message, str)
```

Reply:
fixed